CONTRIBUTING.md

```diff
@@ -1,7 +1,15 @@
-# Contributing to Akka
+# Welcome! Thank you for contributing to Akka!
+
+We follow the standard GitHub [fork & pull](https://help.github.com/articles/using-pull-requests/#fork--pull) approach to pull requests. Just fork the official repo, develop in a branch, and submit a PR!
+
+You're always welcome to submit your PR straight away and start the discussion (without reading the rest of this wonderful doc, or the README.md). The goal of these notes is to make your experience contributing to Akka as smooth and pleasant as possible. We're happy to guide you through the process once you've submitted your PR.
```

Comment:
I wonder if we do not say further down that you should always open a ticket first, so this could be a bit contradictory.

Reply:
Hmm indeed we do:
```
1. Make sure an issue exists in the [issue tracker](https://github.com/akka/akka/issues) for the work you want to contribute.
   - If there is no ticket for it, [create one](https://github.com/akka/akka/issues/new) first.
```

Perhaps we should weaken that a bit, something like:
```
1. To avoid duplicated effort, it might be good to check the [issue tracker](https://github.com/akka/akka/issues) and [existing pull requests](https://github.com/akka/akka/pulls) for existing work.
  - If there is no ticket yet, feel free to [create one](https://github.com/akka/akka/issues/new) to discuss the problem and the approach you want to take to solve it.
```
?

Reply:
Your rewording proposal sounds good @raboof